README.md

```diff
@@ -15,7 +15,7 @@ For those intending to be users of funtofem and not developers, the easiest way
 Conda packages of funtofem are available for the Linux and Mac OS from our smdogroup channel. The user should first open a terminal and create
 a conda environment, such as `F2F`, and then install funtofem as follows with conda install.
 ```
-conda create -n F2F python=3.8
+conda create -n F2F python=3.11
```

Comment:
Why are you setting the python version to 3.11 here for the FUNtoFEM anaconda environment if we're testing against 3.9 now?

Reply:
Users should be allowed to use any version of Python we support, not just the one we test. I only made this 3.11 so we don't have to update the readme instructions for a while.

Reply:
We also test against all supported python versions during conda package deployment, so technically all supported versions are tested.

Reply:
I would prefer consistency across the versions specified here; I don't think it's a big change for the readme. I was referring to version number within the .github workflows specifically.

Reply:
The README should now reference Python 3.9